src/object/C4Id.h

```diff
@@ -34,13 +34,13 @@ class C4ID
 	static NamesList names;
 	void assign(const std::string &s);
 	template<size_t N>
-	explicit C4ID(const char (&s)[N]) { assign(s); }
+	explicit C4ID(const char (&s)[N]) { assign(s); } // TODO: v is not assigned
 public:
 	static const C4ID None; // Invalid ID
-		DEPRECATED static const C4ID Flag;
+	DEPRECATED static const C4ID Flag;
```

Comment:
Are these still used anywhere? We should just remove them otherwise...

Reply:
Flag and Conkit are not used, Melee is used by C4SGame::IsMelee(), but imo it can be removed there, because we no longer have such an ID. The other solution is updating the ID to the current melee goal.

Reply:
Whatever the solution, I'd merge the PR first and then remove the deprecated IDs on master.